openml/runs/functions.py

```diff
@@ -123,8 +123,14 @@ def run_flow_on_task(flow, task, avoid_duplicate_runs=True, flow_tags=None,
     if not isinstance(flow.flow_id, int) or flow_id == False:
         _publish_flow_if_necessary(flow)
 
-    run = OpenMLRun(task_id=task.task_id, flow_id=flow.flow_id,
-                    dataset_id=dataset.dataset_id, model=flow.model, tags=tags)
+    run = OpenMLRun(
+        task_id=task.task_id,
+        flow_id=flow.flow_id,
+        dataset_id=dataset.dataset_id,
+        model=flow.model,
+        tags=tags,
+        flow_name=flow.name,
```

Comment:
This is the only difference, right?

Reply:
Correct, the rest is just reformatting to make such changes look nicer in the future.